### PR TITLE
Changed CPUInfo recipe to build v0.0.20200228

### PR DESCRIPTION
**Pull request by stemann**
Referenced by Torch v1.6.0 - v1.7.1: https://github.com/pytorch/pytorch/tree/v1.7.1/third_party
via XNNPACK: https://github.com/google/XNNPACK/blob/1b354636b5942826547055252f3b359b54acff95/cmake/DownloadCpuinfo.cmake

### CODE REVIEW

File: C/CPUInfo/build_tarballs.jl [hunk 1]
```diff
@@ -37,6 +41,7 @@ fi
 # platforms are passed in on the command line
 platforms = supported_platforms()
 filter!(p -> !(Sys.isapple(p) && arch(p) == "aarch64"), platforms) # aarch64-macos unsupported
+filter!(p -> !(Sys.iswindows(p) && arch(p) == "i686"), platforms) # Failing to link
```
Comment (giordano) [inline C/CPUInfo/build_tarballs.jl:44]: What's the problem?

Comment (giordano) [reply C/CPUInfo/build_tarballs.jl:44]:
```
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x23): undefined reference to `InitOnceExecuteOnce'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x9b): undefined reference to `GetMaximumProcessorGroupCount'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0xdc): undefined reference to `GetMaximumProcessorCount'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x18b): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x271): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x2de): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x48d): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] collect2: error: ld returned 1 exit status
[19:27:56] make[2]: *** [CMakeFiles/cpuinfo.dir/build.make:322: libcpuinfo.dll] Error 1
```
According to https://docs.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-initonceexecuteonce, `InitOnceExecuteOnce` needs linking to `kernel32.dll`

Comment (stemann) [reply C/CPUInfo/build_tarballs.jl:44]: Its fixed in the earlier released subsequent version 20200612, #4560, and again in #4617.

Comment (stemann) [reply C/CPUInfo/build_tarballs.jl:44]: … so maybe not worth the trouble?

Comment (giordano) [reply C/CPUInfo/build_tarballs.jl:44]: Ok, then I'll go ahead and merge this :+1:

**Pull request merged**